.obs/chartfile/values.yaml

```diff
@@ -9,6 +9,10 @@ seedImage:
   tag: "%VERSION%"
   imagePullPolicy: IfNotPresent
 
+channel:
+  repository: "%%IMG_REPO%%/rancher/elemental-teal-channel/5.3" 
+  tag: latest
```

Comment:
🤔 wondering if using latest here can cause issues in the future: but guess no, as we will *add* more versions usually, right?

Reply:
Good point, I guess we should make sure to include a pull policy to always pull the last image. This is expected to mutate as there is a polling logic associated.

Reply:
with the "latest" tag the kubelet should always download by default, so we should be safe